test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.Test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.Test.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.2</TargetFramework>
+    <AssemblyName>Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.Test</AssemblyName>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="**\*.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk">
+      <Version>15.0.0-preview-*</Version>
```

Comment:
Recommend hard coding this to `15.0.0-preview-20161123-03`. The VSTest team has decided (for some reason) to publish nighlies to nuget.org instead of myget.org. They're package is pretty volatile right now.